worker-taskstowing/src/main/java/com/github/jobservice/workers/taskstowing/database/StowedTaskDAO.java

```diff
@@ -0,0 +1,105 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.jobservice.workers.taskstowing.database;
+
+import java.util.List;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.Define;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.sqlobject.statement.SqlBatch;
+import static com.github.jobservice.workers.taskstowing.database.StowedTaskColumnName.*;
+
+interface StowedTaskDAO
+{
+    @SqlUpdate("insert into <table> "
+        + "("
+        + PARTITION_ID + ", "
+        + JOB_ID + ", "
+        + TASK_CLASSIFIER + ", "
+        + TASK_API_VERSION + ", "
+        + TASK_DATA + ", "
+        + TASK_STATUS + ", "
+        + CONTEXT + ", \""
+        + TO + "\", "
+        + TRACKING_INFO_JOB_TASK_ID + ", "
+        + TRACKING_INFO_LAST_STATUS_CHECK_TIME + ", "
+        + TRACKING_INFO_STATUS_CHECK_INTERVAL_MILLIS + ", "
+        + TRACKING_INFO_STATUS_CHECK_URL + ", "
+        + TRACKING_INFO_TRACKING_PIPE + ", "
+        + TRACKING_INFO_TRACK_TO + ", "
+        + SOURCE_INFO + ", "
+        + CORRELATION_ID + ") values "
+        + "(:partitionId, :jobId, :taskClassifier, :taskApiVersion, :taskData, :taskStatus, :context, :to, :trackingInfoJobTaskId, "
+        + ":trackingInfoLastStatusCheckTime, :trackingInfoStatusCheckIntervalMillis, :trackingInfoStatusCheckUrl, "
+        + ":trackingInfoTrackingPipe, :trackingInfoTrackTo, :sourceInfo, :correlationId)")
+    void insertStowedTask(@Define("table") final String table,
+                          @Bind("partitionId") final String partitionId,
+                          @Bind("jobId") final String jobId,
+                          @Bind("taskClassifier") final String taskClassifier,
+                          @Bind("taskApiVersion") final Integer taskApiVersion,
+                          @Bind("taskData") final byte[] taskData,
+                          @Bind("taskStatus") final String taskStatus,
+                          @Bind("context") final byte[] context,
+                          @Bind("to") final String to,
+                          @Bind("trackingInfoJobTaskId") final String trackingInfoJobTaskId,
+                          @Bind("trackingInfoLastStatusCheckTime") final Long trackingInfoLastStatusCheckTime,
+                          @Bind("trackingInfoStatusCheckIntervalMillis") final Long trackingInfoStatusCheckIntervalMillis,
+                          @Bind("trackingInfoStatusCheckUrl") final String trackingInfoStatusCheckUrl,
+                          @Bind("trackingInfoTrackingPipe") final String trackingInfoTrackingPipe,
+                          @Bind("trackingInfoTrackTo") final String trackingInfoTrackTo,
+                          @Bind("sourceInfo") final byte[] sourceInfo,
+                          @Bind("correlationId") final String correlationId);
```

Comment:
This is one of the few places where you shouldn't really use `final`.  There is no implementation so `final` has no meaning in interface definitions (it's sort of surprising that it's allowed).